dev/com.ibm.websphere.appserver.features/visibility/auto/com.ibm.ws.jaxrs2.1-appSecurity3.0.feature

```diff
@@ -0,0 +1,14 @@
+-include= ~${workspace}/cnf/resources/bnd/feature.props
+symbolicName=com.ibm.ws.jaxrs2.1-appSecurity3.0
+visibility=private
+IBM-App-ForceRestart: install, \
+ uninstall
+Subsystem-Version: 1.1.0
+IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(|(osgi.identity=com.ibm.websphere.appserver.jaxrs-2.1)))", \
```

Comment:
@jhanders34 can you check this? I'm not fluent in the provisioning syntax

Reply:
```suggestion
IBM-Provision-Capability: osgi.identity; filter:="(&(type=osgi.subsystem.feature)(osgi.identity=com.ibm.websphere.appserver.jaxrs-2.1))", \
```